scripts/codebuild

```diff
@@ -6,30 +6,67 @@ set -x
 echo "${DOCKER_PASSWORD}" | \
   docker login --username "${DOCKER_USERNAME}" --password-stdin
 
-#LOCUSTFILE can be set in the build overrides of the CodeBuild config
-docker build -t "${ECR_REPO}:exp" --build-arg locustfile="${LOCUSTFILE}" .
+tag=${CODEBUILD_RESOLVED_SOURCE_VERSION:-exp}
+image="${ECR_REPO}:${tag}"
 
-aws ecr get-login-password --region $AWS_REGION \
+docker build -t "${image}" .
+
+aws ecr get-login-password --region "${AWS_REGION}" \
   | docker login --username AWS --password-stdin "${ECR_REPO}"
 
-docker push "${ECR_REPO}:exp"
+docker push "${ECR_REPO}:${tag}"
 
 #This will return an inactive task definition if a new one has been published
 #In that scenario you can specify the new revision manually in update-service
 current_task_definition=$(aws ecs describe-services \
                               --cluster loadtesting \
-                              --service exp-web | \
+                              --service exp-web)
+
+task_definition_name=$(echo "${current_task_definition}" | \
                             jq  '.services[0].taskDefinition' | \
                             cut -d/ -f 2 | \
                             tr -d '"')
 
-#This has not been replacing the current service so you may need to stop the
-#previous deployment until we can figure out why --force-new-deployment is
-#required for the same task definition revision.
+taskDefinitionKeys=(containerDefinitions
+                    family
+                    taskRoleArn
+                    executionRoleArn
+                    networkMode
+                    volumes
+                    placementConstraints
+                    cpu
+                    memory)
+
+joinedKeys=$(printf ",%s" "${taskDefinitionKeys[@]}")
+# remove leading comma
+joinedKeys=${joinedKeys:1}
+
+jq_filter=".taskDefinition | {${joinedKeys}}"
+
+raw_task_definition=$(aws ecs describe-task-definition \
+                          --task-definition "${task_definition_name}" | \
+                        jq "${jq_filter}")
+
+# LOCUSTFILE can be set in the build overrides of the CodeBuild config
+: "${LOCUSTFILE:=/app/locustfiles/prime.py}"
```

Comment:
I had to look up what `:` means for shell scripting, I guess it's just a noop value of true?

Reply:
Sorry, yeah, this is a convention I've seen elsewhere. When using `:=` variable substitution, that statement doesn't do anything but modify the variable, so using `:` is the way to indicate that so shellcheck doesn't complain.

Is there a comment or something I should add?

Reply:
I added a comment here, too